tests/queries/0_stateless/01400_join_get_with_multi_keys.sql

```diff
@@ -6,4 +6,6 @@ INSERT INTO test_joinGet VALUES ('ab', '1', 0.1), ('ab', '2', 0.2), ('cd', '3',
 
 SELECT joinGet(test_joinGet, 'c', 'ab', '1');
 
+SELECT joinGet(join_test, 'b', toLowCardinality('hellohellohellohell\0hellohell\0hellohellohellohellohellohell\0hellohell\0hellohellohellohellohellohell\0hellohell\0hellohellohellohellohellohell\0hellohell\0hellohello'), 1); -- { serverError 60 }
```

Comment:
Maybe add a positive test that it actually works for LowCardinality?

Reply:
done